webpack/sources.js

```diff
@@ -1,8 +1,6 @@
 var config = require('config');
 
 module.exports = function getEntrySources(sources) {
```

Comment:
You can remove this file and simply pass the option `--inline` to the webpack-dev-server command.